### ISSUE TITLE
Not able to view any anime

### ISSUE THREAD

**Issue by itsmemac** [XenTeckzX/FireAnime#5]
Tried watching anime using the latest version of the apk, it always send me to fireanime home after i select the anime episode.

**Comment by XenTeckzX** [comment]
which anime you trying to watch?

**Comment by itsmemac** [comment]
Any animes
and the app is very slow 
it take a minute or 2 to select stream button in the anime

**Comment by XenTeckzX** [comment]
what device are you using? since on the Fire stick it loads very fast

**Comment by itsmemac** [comment]
Have been using this in fire tv stick since start but this issue started happening just last update.
Fire Stick tv is up to date

**Comment by XenTeckzX** [comment]
is it slow when you reach this page https://i.redd.it/d2ysg2u0y8o11.jpg or is it slow when your selecting an episode to watch

**Comment by XenTeckzX** [comment]
I think the issue may be the stream checker to check for dead links that i have added if your having problems looking for sources i could add something to disable it which should make sources load faster *BUT* some sources may be broken without it

**Comment by itsmemac** [comment]
any fix yet?

**Comment by XenTeckzX** [comment]
gonna send you a release of a private build if you can test that out it would be great

**Comment by itsmemac** [comment]
> gonna send you a release of a private build if you can test that out it would be great

Sure. Send me those. Will test it and keep you posted

**Comment by XenTeckzX** [comment]
Here's the link https://github.com/XenTeckzX/FireAnime/raw/master/FireAnimeCustomTestBuild.apk

in this version i've removed the stream checker so in theory you don't have to wait for all the streams to load up

**Comment by itsmemac** [comment]
> Here's the link https://github.com/XenTeckzX/FireAnime/raw/master/FireAnimeCustomTestBuild.apk
> 
> in this version i've removed the stream checker so in theory you don't have to wait for all the streams to load up

Still the same :(

**Comment by XenTeckzX** [comment]
does the anime episodes show up,
Does the anime page look like this https://i.imgur.com/e9vD46W.jpg
with no description

**Comment by itsmemac** [comment]
just like the screenshot
Can't even navigate to the stream button, only can go back to homepage. 
sometimes it stops at stream button (highlight in yellow) and when I press stream, 
it shows episode and when I click any episode it sends me back to homepage.

**Comment by XenTeckzX** [comment]
try changing your date and time settings it seems to affect the application

**Comment by itsmemac** [comment]
same. no change

**Comment by Sidh-27** [comment]
i'm also actually having the same problem but it loaded properly for me only once . please fix this , i really like this app, thanks

**Comment by Nuamanwt** [comment]
Same here

**Comment by smonsels** [comment]
Im having the same issue,
I'm using the latest Fire Anime app and I did update but for some odd reason when I go to wco or animepahe  all of the anime that are there when you choose and click on it, all of the episodes lists does NOT show, I so i deleted the app, clear the entire data,  & cache then reinstall it everything was good to go, then turn shut down the first goe back to my regular tv then 8 min later decide to go back on the lists dissappear again, idk what is goin on, plase help!
Everything is up to date on my firestick

**Comment by RetroGamer2020** [comment]
i Agree the Select Player option doesn't show up so i can't watch anime fix this issue in the upcoming patch
